web/src/features/menu/list/index.tsx

```diff
@@ -59,41 +61,71 @@ const ListMenu: React.FC = () => {
         });
         break;
       case 'ArrowRight':
-        if (!Array.isArray(menu.items[selected].values)) return;
-        setIndexStates({
-          ...indexStates,
-          [selected]:
-            indexStates[selected] + 1 <= menu.items[selected].values?.length! - 1
-              ? indexStates[selected] + 1
-              : (indexStates[selected] = 0),
-        });
+        if (Array.isArray(menu.items[selected].values)) {
+          setIndexStates({
+            ...indexStates,
+            [selected]:
+              indexStates[selected] + 1 <= menu.items[selected].values?.length! - 1
+                ? indexStates[selected] + 1
+                : 0
+          });
+        } else if (menu.items[selected].checked !== undefined) {
+          setCheckedStates({
+            ...checkedStates,
+            [selected]: !checkedStates[selected]
+          });
+        }
         break;
       case 'ArrowLeft':
-        if (!Array.isArray(menu.items[selected].values)) return;
-        setIndexStates({
-          ...indexStates,
-          [selected]:
-            indexStates[selected] - 1 >= 0
-              ? indexStates[selected] - 1
-              : (indexStates[selected] = menu.items[selected].values?.length! - 1),
-        });
+        if (Array.isArray(menu.items[selected].values)) {
+          setIndexStates({
+            ...indexStates,
+            [selected]:
+              indexStates[selected] - 1 >= 0
+                ? indexStates[selected] - 1
+                : menu.items[selected].values?.length! - 1
+          });
+        } else if (menu.items[selected].checked !== undefined) {
+          setCheckedStates({
+            ...checkedStates,
+            [selected]: !checkedStates[selected]
+          });
+        }
```

Comment:
I don't know how I feel about left-right arrows checking/unchecking the checkbox, imo this should be handled by `Enter`, then just make it not close the menu in the process.

While left-right arrows to check/uncheck are cool it adds unnecessarily complexity to the code and most people's first instinct would be to press enter to toggle the checkbox.

Reply:
Well, both enter and left/right handle it

Reply:
yes but not returning out of the confirm function doesn't give the user that feedback that the checkbox changed

Reply:
We've got the onCheck function for that